llama_index/indices/document_summary/base.py

```diff
@@ -223,16 +223,41 @@ def _insert(self, nodes: Sequence[BaseNode], **insert_kwargs: Any) -> None:
         self._add_nodes_to_index(self._index_struct, nodes)
 
     def _delete_node(self, node_id: str, **delete_kwargs: Any) -> None:
-        """Delete a node."""
-        if node_id not in self._index_struct.doc_id_to_summary_id:
-            raise ValueError(f"node_id {node_id} not in index")
-        summary_id = self._index_struct.doc_id_to_summary_id[node_id]
+        pass
 
-        # delete summary node from docstore
-        self.docstore.delete_document(summary_id)
+    def delete_nodes(
```

Comment:
deleting individual nodes is requires a bit more logic. You need to check if you deleted all nodes for a given summary/doc id, and if so, delete that summary from the docstore and vector store